presto-jmx/src/main/java/io/prestosql/plugin/jmx/JmxMetadata.java

```diff
@@ -96,6 +87,22 @@ public JmxTableHandle getTableHandle(ConnectorSession session, SchemaTableName t
         return getTableHandle(tableName);
     }
 
+    @Override
+    public boolean usesLegacyTableLayouts()
+    {
+        return false;
+    }
+
+    @Override
+    public ConnectorTableProperties getTableProperties(ConnectorSession session, ConnectorTableHandle table)
+    {
+        return new ConnectorTableProperties(TupleDomain.all(),
+            Optional.empty(),
+            Optional.empty(),
+            Optional.empty(),
+            emptyList());
```

Comment:
2 formatting nits (somehow this comment got lost when i first wrote it)

- per code style, when each param is on a separate line, there should be a newline before the first one:
  ```
   return new ConnectorTableProperties(
       TupleDomain.all(),
   ```
- by convention, we use `ImmutableList.of()` rather than `emptyList()` (i understand this is subjective, this is just the convention we try to adhere to, as we maintain codebase consistency)

Reply:
Following the convention of project makes sense. I fixed them.